bitmagnet.io/setup/installation.md

```diff
@@ -61,6 +59,26 @@ services:
 
 After running `docker compose up -d` you should be able to access the web interface at `http://localhost:3333`. The DHT crawler should have started and you should see items appear in the web UI within around a minute.
 
+## TrueNas Scale
+
+To effortlessly kickstart your journey with bitmagnet, integrate the [TrueCharts catalog](https://truecharts.org/manual/SCALE/guides/getting-started#adding-truecharts) and proceed with app installation on TrueNAS Scale.
+
+Optionally, you can configure the `TMDB API Key` by following the guidelines outlined [here](configuration.md#obtaining-a-tmdb-api-key).
+
+![Bitmagnet Config Options](/assets/images/bitmagnent-scale-app-config.png)
+
+Worth noting, the PostgreSQL database is seamlessly configured and installed alongside the app. Just ensure to meticulously adhere to the getting started guide, installing all the recommended operators before initiating the app installation process.
+
+![Bitmagnet Scale App](/assets/images/bitmagnet-scale-app.png)
```

Comment:
This is a bit of a "useless" screenshot.
Taking a lot of space on someone elses installation page.